src/components/AuthenticatedAppHeader/SightingsButton.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+
+import useSightingTermQuery from '../../models/sighting/useSightingTermQuery';
+import Text from '../Text';
+import SearchButton from './SearchButton';
+import SearchResult from './SearchResult';
+
+export default function SightingsButton() {
+  const [inputContent, setInputContent] = useState('');
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const {
+    data: searchResults,
+    loading,
+    error,
+  } = useSightingTermQuery(searchTerm);
+
+  const resultsCurrent = inputContent === searchTerm;
+  const noResults = searchResults && searchResults.length === 0;
```

Comment:
different from searchResults?.length === 0
?

Reply:
<img width="272" alt="image" src="https://user-images.githubusercontent.com/9775178/163048025-d5a93efb-b9f7-4b87-b16e-6bfd8a801e94.png">
this was the case i was thinking about.. though i suppose the result is the same so maybe i can switch it!